stancon_2020/InferenceData_Stancon2020_fi.ipynb

```diff
@@ -0,0 +1,4499 @@
+{
```

Comment:
<p>This should be fixed I think right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>